logger/zap/zap.go

```diff
@@ -10,8 +10,8 @@ type Logger struct {
 }
 
 func New() logger.Logger {
-	logger, _ := zap.NewProduction(zap.AddCallerSkip(1))
-	sugar := logger.Sugar()
+	log, _ := zap.NewProduction(zap.AddCallerSkip(1))
```

Comment:
Why skip error here?

Reply:
if return an error with logger it will break API

Reply:
I think that basing program should decide what to do with the error, not package, if it is not initialized, then there will be a null pointer error inside the package, which is bad.
And the API just will need to be updated

Reply:
@pcherednichenko what will you do when somehow you cannot init logger? :)

Reply:
@m1ome we can use this logger, for example in serious financial projects, then we should not ignore the error, but in a some funny projects we can

Reply:
@m1ome @pcherednichenko I vote for make such changes with error returning in the next version